src/abstract-view.js

```diff
@@ -201,11 +201,9 @@ var AbstractView = Backbone.View.extend({
   // DOM and unbinding it. Regions will call this method
   // for you. You can specify an `onDestroy` method in your view to
   // add custom code that is called after the view is destroyed.
-  destroy: function() {
```

Comment:
we should apply this exact same thing to `Object#destroy` which explicitly passes `options` instead of the `args`